infra/src/main/kotlin/fi/fta/geoviite/infra/ratko/RatkoController.kt

```diff
@@ -44,9 +44,9 @@ class RatkoController(private val ratkoServiceParam: RatkoService?, private val
 
     @PreAuthorize(AUTH_VIEW_LAYOUT)
     @PostMapping("/push-location-tracks")
-    fun pushLocationTracksToRatko(@RequestBody changes: List<LocationTrackChange>): ResponseEntity<String> {
+    fun pushLocationTracksToRatko(@RequestBody changes: List<LocationTrackChange>): ResponseEntity<Unit> {
```

Comment:
Meillä on enimmäkseen kaikki post pyynnöt palauttaneet jotain, mutta koodin seasta löytyy myös muutama muu paikka jossa ei haluta palauttaa sisältöä vaan pelkkä http status koodi. Tuosta on muutamaakin eri varianttia. Nämä voisi varmaaan toteuttaa kaikki samalla tavalla (ja varmistaa että toimiiko nuo vanhatkaan, koska osa noista on vähän suspect). Nämä pari esim löytyy samaisesta ratkocontrollerista peräkäin:
```
    @PreAuthorize(AUTH_EDIT_LAYOUT)
    @PostMapping("/push-designs")
    fun pushDesignChangesToRatko(): HttpStatus {
        // TODO implement
        return HttpStatus.NO_CONTENT
    }

    @PreAuthorize(AUTH_VIEW_LAYOUT)
    @PostMapping("/push-location-tracks")
    fun pushLocationTracksToRatko(@RequestBody changes: List<LocationTrackChange>): ResponseEntity<String> {
        ratkoService.pushLocationTracksToRatko(LayoutBranch.main, changes)
        return ResponseEntity(HttpStatus.OK)
    }
```

Tuosta pitäisi testata että palauttaako spring statuskoodin oikein noilla malleilla vai ei ja voisi vetää kaikki samaan malliin. Sinällään tämä sinun versio näyttää kyllä ainakin varsin eksplissiittiseltä ja uskoisin että tästä tulee tyhjä body + oikea koodi väkisin. Mutta toisaalta jos pelkkä `return HttpStatus.NO_CONTENT` tekisi saman, se olisi toki hiukan lyhyempi.

Reply:
Mielestäni testailin noita malleja tässä yhteydessä ja tuo ylempi quotaamasi itse asiassa ei muistaakseni tee sitä mitä sen olettaisi. Jos endpointista palauttaa statuskoodin, se muistaakseni palautuu `OK 200`:na siten, että vastauksen bodyssä lukee `204`. Voin tosin vielä pöyhäistä tuota vielä uudestaan review-korjausten yhteydessä, mutta muistaakseni menee näin

Reply:
Joo tuo mitä ei ole toteutettu voi kyllä pistää heittämään, se on varmaan parempikin